client/src/pages/demo/components/index.tsx

```diff
@@ -0,0 +1,22 @@
+import React, { Fragment } from 'react';
+import Dropdown from '@/src/shared/components/Dropdown';
+
+const Components: React.FC = () => {
+  return (
+    <Fragment>
+      <div className="p-4 border-b border-black">
+        <Dropdown />
+        <div className="mt-[5px]">
+          <h1>Props: </h1>
```

Comment:
- please add anchor tag para dali ra e-navigate.
```suggestion
      <div className="p-4 border-b border-black">
        <a href="#dropdown">Dropdown</a>
        <Dropdown />
        <div className="mt-[5px]">
          <h1>Props: </h1>
```